blog/2023-11-01-feature-flags-sdks-architectures.md

```diff
@@ -0,0 +1,159 @@
+---
+slug: 'feature-flags-sdks-architectures'
+title: 'Feature Flags services Server-side SDKs different architectures approaches'
+date: 2023-11-01
+authors: [liran2000]
+description: "Feature Flags services Server-side SDKs different architectures approaches"
+tags: [client, sdk]
+draft: false
+---
+
+Server-side feature flag software development kits (SDKs) are common for letting applications like microservices interact with Feature flag services.
+Feature Flags SDKs have several used functionalities, mainly feature flag evaluation with or without context.
```

Comment:
change "Flags" to "flag" and remove "used" to read:
Feature flag SDKs have several functionalities, mainly feature flag evaluation with or without context.